lib/Model/Importer/DebugInfo/PDBImporter.cpp

```diff
@@ -0,0 +1,1089 @@
+  }
+
+  return Error::success();
+}
+
+// Parse LF_MEMBER.
+Error PDBImporterTypeVisitor::visitKnownMember(CVMemberRecord &Record,
+                                               DataMemberRecord &Member) {
+  InProgressMemberTypes[CurrentTypeIndex].push_back(Member);
+
+  return Error::success();
+}
+
+llvm::Error
+PDBImporterTypeVisitor::visitKnownRecord(CVType &CVR,
+                                         MemberFunctionRecord &MemberFnRecord) {
+  InProgressConcreteFunctionMemberTypes[CurrentTypeIndex] = MemberFnRecord;
+  return Error::success();
+}
+
+// Parse LF_ONEMETHOD.
+// This occurs within LF_CLASS and it references an LF_MFUNCTION.
+Error PDBImporterTypeVisitor::visitKnownMember(CVMemberRecord &Record,
+                                               OneMethodRecord &FnMember) {
+  InProgressFunctionMemberTypes[CurrentTypeIndex].push_back(FnMember);
+  return Error::success();
+}
+
+// Parse LF_ENUMERATE.
+Error PDBImporterTypeVisitor::visitKnownMember(CVMemberRecord &Record,
+                                               EnumeratorRecord &Member) {
+  InProgressEnumeratorTypes[CurrentTypeIndex].push_back(Member);
+  return Error::success();
+}
+
+// LF_CLASS, LF_STRUCTURE, LF_INTERFACE (TPI)
+Error PDBImporterTypeVisitor::visitKnownRecord(CVType &Record,
+                                               ClassRecord &Class) {
+  using namespace model;
+  // 0-sized structs are typedefed to void.
+  if (not Class.getSize()) {
+    auto TypedefTy = makeType<TypedefType>();
+    TypedefTy->OriginalName = Class.getName();
+
+    auto TypeDef = cast<TypedefType>(TypedefTy.get());
+    using Values = model::PrimitiveTypeKind::Values;
+    TypeDef
+      ->UnderlyingType = QualifiedType(Model->getPrimitiveType(Values::Void, 0),
+                                       {});
+
+    auto TyPath = Model->recordNewType(std::move(TypedefTy));
+
+    ProcessedTypes[CurrentTypeIndex] = TyPath;
+
+    return Error::success();
+  }
+
+  TypeIndex FieldsTypeIndex = Class.getFieldList();
+  if (InProgressMemberTypes.count(FieldsTypeIndex)) {
+    auto NewType = makeType<model::StructType>();
+    NewType->OriginalName = Class.getName();
+    auto Struct = cast<model::StructType>(NewType.get());
+
+    Struct->Size = Class.getSize();
+    auto &TheFields = InProgressMemberTypes[FieldsTypeIndex];
+    uint64_t MaxOffset = 0;
+
+    for (const auto &Field : TheFields) {
+      // Create new field.
+      uint64_t Offset = Field.getFieldOffset();
+      if (not ProcessedTypes.count(Field.getType())
+          and Field.getType().isSimple())
+        createPrimitiveType(Field.getType());
+
+      if (ProcessedTypes.count(Field.getType())) {
+        auto MaybeSize = ProcessedTypes[Field.getType()].get()->size();
+        uint64_t Size = 0;
+        if (not MaybeSize)
+          Size = 0;
+        else
+          Size = *MaybeSize;
+
+        if (Size == 0) {
+          // Skip 0-sized field.
+          continue;
```

Comment:
Every time you ignore something from the PDB, `revng_log` about it. Not importing stuff is OK, but let's not let them happen silently.
Please ensure this happens everywhere.